spec/migrations/20150625220141_fix_serialized_reports_for_rails_four_spec.rb

```diff
@@ -1,9 +1,25 @@
 require_migration
 
 describe FixSerializedReportsForRailsFour do
+  before do
+    # Create the various classes that are represented in the serialized data
+    class ::MiqReport; end
+    class ::Ruport
+      class Data
+        class Table; end
+        class Record; end
+      end
+    end
+  end
+
+  after do
+    Object.send(:remove_const, "MiqReport")
+    Object.send(:remove_const, "Ruport")
+  end
+
   let(:report_result_stub)  { migration_stub(:MiqReportResult) }
   let(:binary_blob)  { migration_stub(:BinaryBlob) }
-  let(:data_dir) { File.join(Rails.root, 'spec/migrations/data', File.basename(__FILE__, '.rb')) }
+  let(:data_dir) { File.expand_path(File.join("data", File.basename(__FILE__, '.rb')), __dir__) }
```

Comment:
Don't think I didn't see that typo :trollface:
![image](https://cloud.githubusercontent.com/assets/651659/25755719/0f05ea2c-3192-11e7-98f1-5fc2bfc059c1.png)